tutorial/opt/aero/plot_optHist.py

```diff
@@ -3,49 +3,92 @@
 import matplotlib.pyplot as plt
 from pyoptsparse import History
 
-plt.rcParams["text.usetex"] = True  # Comment out if latex installation is not present
-plt.rcParams["font.family"] = "serif"
-plt.rcParams["font.size"] = 20
-plt.rcParams["xtick.labelsize"] = 16
-plt.rcParams["ytick.labelsize"] = 16
-plt.rcParams["lines.linewidth"] = 3.0
 
-parser = argparse.ArgumentParser()
-parser.add_argument("--histFile", type=str, default="opt.hst")
-parser.add_argument("--outputDir", type=str, default="./")
-args = parser.parse_args()
+def main():
 
-optHist = History(args.histFile)
-histValues = optHist.getValues()
+    plt.rcParams["text.usetex"] = True  # Comment out if latex installation is not present
```

Comment:
These typically live outside of `main()`

Reply:
I have removed the plt.rcParams and parser lines of code outside the main function